slack/web/classes/actions.py

```diff
@@ -0,0 +1,207 @@
+from abc import ABCMeta, abstractmethod
+from typing import Iterable, Set, Union
+
+from .objects import (
+    ButtonStyles,
+    DynamicDropdownTypes,
+    EnumValidator,
+    JsonObject,
+    JsonValidator,
+    Option,
+    OptionGroup,
+    extract_json,
+)
+
+
+class Action(JsonObject):
+    attributes = {"name", "text", "type", "url"}
+
+    def __init__(self, *, text: str, type: str, name: str = None, url: str = None):
```

Comment:
I followed the larger lead of the library here and switched everything to require kwargs - not sure if that's something we want these classes to do or not